UnitTests/PaymentButtonsTests/PayPalCreditButton_Tests.swift

```diff
@@ -14,7 +14,7 @@ class PayPalCreditButton_Tests: XCTestCase {
         let sut = PayPalCreditButton()
         XCTAssertEqual(sut.edges, PaymentButtonEdges.softEdges)
         XCTAssertEqual(sut.size, PaymentButtonSize.collapsed)
-        XCTAssertEqual(sut.color, PaymentButtonColor.darkBlue)
+        XCTAssertEqual(sut.color, PaymentButtonColor.white)
```

Comment:
Just confirming, should white or gold be the default color?

Reply:
Good point, I just checked Figma and gold is the recommended/default color. Will make that update :)